coil-core/src/commonTest/kotlin/coil3/UriTest.kt

```diff
@@ -39,6 +39,17 @@ class UriTest {
         assertEquals("something:/test/relative/image.jpg", uri.fragment)
     }
 
+    @Test
+    fun windows() {
+        val uri = "D:\\test\\relative\\image.jpg".toUri()
+        assertNull(uri.scheme)
```

Comment:
Is it correct that scheme is null? Should it be `D:`?

Reply:
I think D is the volume information, it should be part of the path.

Reply:
> Is it correct that scheme is null? Should it be `D`?

I think D is the volume information, it should be part of the path.